test/unit/auth/auth-config.spec.ts

```diff
@@ -769,6 +783,27 @@ describe('OIDCConfig', () => {
       expect(config.enabled).to.be.true;
     });
 
+    it('should set readonly property clientSecret', () => {
+      expect(config.clientSecret).to.equal('CLIENT_SECRET');
+    });
+
+    it('should set readonly property expected responseType', () => {
+      const expectedResponseType = {
+        code: true,
+      };
+      expect(config.responseType).to.deep.equal(expectedResponseType);
+    });
+
+    it('should set readonly property default responseType', () => {
+      delete serverResponse.clientSecret;
```

Comment:
serverResponse is a variable shared across many test cases. Deep copy it before making changes to it.